app/assets/src/components/common/DetailsSidebar/BulkDownloadDetailsMode/AdvancedDownloadTab.jsx

```diff
@@ -43,13 +43,11 @@ export default class AdvancedDownloadTab extends React.Component {
       return "Failed to generate command. Please contact us for help.";
     }
 
-    return `curl -L "${bulkDownload.presigned_output_url}" > "${
-      bulkDownload.download_name
-    }.tar.gz"\
-        && mkdir -p "${bulkDownload.download_name}"\
-        && tar -zvxf "${bulkDownload.download_name}.tar.gz" -C "${
-      bulkDownload.download_name
-    }"\
+    let bulkDownloadFileName = `${bulkDownload.download_name} ID-${bulkDownload.id}`;
```

Comment:
What about adding a timestamp instead?
The id might be weird for the user, since it is an internal number that the user never sees.

Reply:
That’s a great idea! Olivia suggested the same thing earlier

Reply:
Here's an example of what the updated changes would output:
"Original Input Files - 05-26-2020 08-47-33am"